app/src/main/java/comcmput301f16t01/github/carrier/Users/UserProfileActivity.java

```diff
@@ -255,4 +281,85 @@ public void hideKeyboard(View v) {
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(v.getWindowToken(), 0);
     }
+
+
+    /**
+     * This is the function that calls the number presented
+     *
+     * @param v view used for this function
```

Comment:
Maybe for this javadoc just specify what view it is (i.e. what are you clicking on to call this function?)